contracts/src/internal/HyperdriveAdmin.sol

```diff
@@ -30,16 +30,16 @@ abstract contract HyperdriveAdmin is IHyperdriveEvents, HyperdriveBase {
         _checkOptions(_options);
 
         // Ensure that the destination is set to the fee collector.
-        address feeCollector = _feeCollector;
+        address feeCollector = _adminController.feeCollector();
```

Comment:
so one side effect of this change is that all feeCollectors, pausers, etc will always be the same for all pools. is that right? if so, is that what we want?

Reply:
Yep, as mentioned offline, I think that this is a practically going to be fine for the foreseeable future as long as it's clearly communicated. For most of the parameters, the address can be set to an access controller or to a contract with a similar architecture to the checkpoint rewarder to support more bespoke functionality.